snap/validate.go

```diff
@@ -356,6 +354,21 @@ func Validate(info *Info) error {
 	return ValidateLayoutAll(info)
 }
 
+// ValidateBase validates the base field.
+func ValidateBase(info *Info) error {
+	// validate that bases do not have base fields
+	if info.Type == TypeOS || info.Type == TypeBase {
+		if info.Base != "" {
```

Comment:
Base can also be "none" for these now, that is also fine

Reply:
Done.